K8s-deployment/Charts/databroker/external-client-service.yaml

```diff
@@ -3,14 +3,14 @@ apiVersion: v1
 metadata:
   annotations:
     # For aws service
-    service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+    #service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
```

Comment:
Can have two files 
1) external-client-aws-service.yaml
2) external-client-azure-service.yaml

Reply:
Split into 2 files for aws and azure and added steps in README to make an `external-client-service.yaml` from these, which is referenced in the install script in the kubectl command.